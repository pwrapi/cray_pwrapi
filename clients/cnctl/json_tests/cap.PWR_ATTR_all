{
 "PWR_Function"          : "PWR_ObjAttrCapabilities",
 "PWR_MajorVersion"      : 32,
 "PWR_MinorVersion"      : 67
}
