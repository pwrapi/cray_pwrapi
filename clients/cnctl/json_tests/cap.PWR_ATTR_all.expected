{
 "PWR_AttrCapabilities"  : [
                            "PWR_ATTR_CSTATE_LIMIT",
                            "PWR_ATTR_FREQ",
                            "PWR_ATTR_FREQ_LIMIT_MAX",
                            "PWR_ATTR_FREQ_LIMIT_MIN"
                           ],
 "PWR_MajorVersion"      : 0,
 "PWR_MinorVersion"      : 1,
 "PWR_ReturnCode"        : 0,
 "PWR_Messages"          : null,
 "PWR_ErrorMessages"     : null
}

